### PR TITLE
Lab 2

### PR DESCRIPTION
**Pull request by Denny-Lymic**

### CODE REVIEW

File: Lab 2.1/CSharp_Net_module1_2_1_lab/LibraryUser.cs [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        //BookInfo() – returns book info by index
+        //BooksCout() – returns current count of books
+
+        public void AddBook(string bookName)
+        {
+            int i = 0;
+            while (bookList[i] != null)
+            {
+                i++;
+                if (i + 1 == BookLimit) return;
```
Comment (Landay7) [inline Lab 2.1/CSharp_Net_module1_2_1_lab/LibraryUser.cs:82]: you can check it before loop

File: Lab 2.1/CSharp_Net_module1_2_1_lab/LibraryUser.cs [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+            }
+            bookList[i] = null;
+            bookCount--;
+        }
+        public int BooksCount() { return bookCount; }
+        public string BookInfo(int index) { return this[index]; }
```
Comment (Landay7) [inline Lab 2.1/CSharp_Net_module1_2_1_lab/LibraryUser.cs:101]: If you have free spaces, bookInfo will not work like that. 
Imagine that you have 7 books. you delete book on 2 positions, then BookInfo(2) should return null?

File: Lab 2.1/CSharp_Net_module1_2_1_lab/LibraryUser.cs (hunk 1 above)
Comment (Denny-Lymic) [reply Lab 2.1/CSharp_Net_module1_2_1_lab/LibraryUser.cs:82]: Why, it should be there, its check for out of range of array

Comment (Landay7) [reply Lab 2.1/CSharp_Net_module1_2_1_lab/LibraryUser.cs:82]: basically, here you check that book count is less than book limit, am I right?

Comment (Denny-Lymic) [reply Lab 2.1/CSharp_Net_module1_2_1_lab/LibraryUser.cs:82]: Yes, "i" is current position in array

**Review by Landay7 — APPROVED** [review]

**Pull request merged**